src/styles/blocks/events.scss

```diff
@@ -0,0 +1,209 @@
+.events {
```

Comment:
![image](https://user-images.githubusercontent.com/92029462/140489194-69419caa-a8c7-4591-ba83-ea88dd6d45ea.png)
Shouldn't these blocks like events or footer center themselves while in mobile mode?